src/components/MenuBar/ButtonBar.js

```diff
@@ -77,13 +81,14 @@ function ButtonBar({ isMobile }) {
 				<PoppinLink to={link.to} key={`nav-${index}`}>
 					{link.name}
 				</PoppinLink>)}
+			{Date.now() > applicationOpen.getTime() &&
 			<BorderLink
-				disabled
```

Comment:
Rather than making the button on the Navigation Bar disappear, let's just disable it.

The original disabled code was this
```
disabled={Date.now() > applyDeadline.getTime()}
```

So modify that as necessary with the application open day